topcoffea/modules/fileReader.py

```diff
@@ -164,7 +164,7 @@ def GetAllInfoFromFile(fname, treeName = 'Events'):
     f = uproot.open(fname)
     t = f[treeName]
     isData = not 'genWeight' in t#.keys()
-    nEvents = len(t['MET_pt'])
+    nEvents = len(t['event'].array())
```

Comment:
Why are we reading in the whole event as numpy arrays just to count the number of entries?  Uproot's `TTree.num_entries` should do the trick.  (See [here](https://uproot.readthedocs.io/en/latest/uproot.behaviors.TTree.TTree.html#num-entries).)  This is how [coffea does it](https://github.com/CoffeaTeam/coffea/blob/529c36e1695ff44f3d69f243572f9ee4f9bcf551/coffea/processor/executor.py#L1219-L1224).  However, this raises the question: If coffea is going to open all of our files and count their entries, why are we also doing it?